types/Issuing/AuthorizationsResource.d.ts

```diff
@@ -119,6 +119,7 @@ declare module 'stripe' {
         /**
          * [Deprecated] Approves a pending Issuing Authorization object. This request should be made within the timeout window of the [real-time authorization](https://stripe.com/docs/issuing/controls/real-time-authorizations) flow.
          * This method is deprecated. Instead, [respond directly to the webhook request to approve an authorization](https://stripe.com/docs/issuing/controls/real-time-authorizations#authorization-handling).
+         * @deprecated This method is deprecated, please refer to the description for details.
```

Comment:
Did we decide to leave these descriptions?

Reply:
Left it for now. Can remove it for in the next release after I merge the codegen changes for it.

Reply:
Can you make sure there is a ticket to track that?